frontend/src/pages/details-page.tsx

```diff
@@ -8,28 +8,32 @@ const DetailsPage = () => {
   const post: Post = state?.post;
   const navigate = useNavigate();
   return (
-    <div className="font-[Poppins]">
-      <div className="relative">
-        <img src={post.imageLink} alt={post.title} className="h-[460px] object-cover w-full " />
+    <div className="w-full font-[Poppins] ">
+      <div className="relative flex flex-col">
+        <img
+          src={post.imageLink}
+          alt={post.title}
+          className="h-[360px] md:h-[460px] object-cover w-full"
+        />
 
         <div className="absolute top-0 left-0 w-full h-full bg-black opacity-50"></div>
-        <div className="px-64 text-white absolute top-16 left-0 cursor-pointer">
-          <FiArrowLeft style={{ fontSize: '36px' }} onClick={() => navigate(-1)} />
+        <div className="top-16 px-4 md:px-8 lg:px-16 md:top-24 w-full justify-start  text-lg md:text-xl lg:text-2xl text-white absolute cursor-pointer">
+          <FiArrowLeft onClick={() => navigate(-1)} />
         </div>
-        <div className="px-64 absolute bottom-16  text-white">
+        <div className=" px-4 md:px-8 lg:px-16 w-full absolute bottom-4 md:bottom-8  lg:bottom-16  text-white">
           <div className="flex space-x-2 mb-6">
             {post.categories.map((category, index) => (
               <div key={index} className="bg-gray-500 text-white rounded-full px-3 py-1 text-sm">
                 {category}
               </div>
             ))}
           </div>
-          <h1 className="text-4xl font-semibold ">{post.title}</h1>
+          <h1 className="text-lg md:text-2xl lg:text-4xl font-semibold ">{post.title}</h1>
         </div>
       </div>
-      <div className="px-64">
+      <div className="px-4 md:px-8 lg:px-16 pt-4 md:pt-8 lg:pt-16  text-left w-full">
         <div className="mt-4">
```

Comment:
This mt-4 adds extra padding, please remove that or adjust accordingly. As per the requirement shown clearly in the image.

Reply:
Margin Have been adjusted